src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/ViewTransferNoteViewModelMap.cs

```diff
@@ -91,6 +91,9 @@ public ViewTransferNoteViewModel Map(ViewTransferNoteViewModelMapTransfer source
                     null),
                 TransferredByAddress = transferredByFormattedAddress,
                 Summary = GenerateNotesModel(source),
+
+                EvidenceNoteHistoryData = null,  //TODO: when transfer history appears at top of view page
```

Comment:
Can be removed a transfer note wont have the history its the original evidence note that has the history. George to update the wireframe.

Reply:
done